src/main/java/rx/observers/TestSubscriber.java

```diff
@@ -227,9 +227,9 @@ public void awaitTerminalEvent() {
      * @throws RuntimeException
      *          if the Subscriber is interrupted before the Observable is able to complete
      */
-    public void awaitTerminalEvent(long timeout, TimeUnit unit) {
+    public boolean awaitTerminalEvent(long timeout, TimeUnit unit) {
```

Comment:
Throw `TimeoutException`? It's a bit minor breaking change. @benjchristensen Is it fine?